applications/StructuralMechanicsApplication/custom_processes/postprocess_eigenvalues_process.cpp

```diff
@@ -0,0 +1,177 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "custom_processes/postprocess_eigenvalues_process.h"
+#include "structural_mechanics_application_variables.h"
+
+
+namespace Kratos
+{
+
+    PostprocessEigenvaluesProcess::PostprocessEigenvaluesProcess(ModelPart &rModelPart,
+                                                                 Parameters OutputParameters) 
+                                                                 : mrModelPart(rModelPart), 
+                                                                   mOutputParameters(OutputParameters)
+    {
+        Parameters default_parameters(R"(
+            {
+                "result_file_name" : "",
+                "animation_steps"   :  1,
+                "use_eigenfrequency_in_label" : false,
```

Comment:
what is the alternative to eigenfrequency in label? Step? Maybe it is clearer to make this a string like `"eigenfrequency_label_type"` that could have values like `"step"`, `"frequency"`, `"angular_frequency"`.

Reply:
Done